mysql-test/include/have_tlsv13.inc

```diff
@@ -0,0 +1,10 @@
+--disable_query_log
+connect (ssl_connection,localhost,root,,,,,SSL);
+
+if (`SELECT VARIABLE_VALUE NOT LIKE 'TLSv1.3' FROM information_schema.GLOBAL_STATUS WHERE VARIABLE_NAME = 'ssl_version'`) {
```

Comment:
Would information_schema.SESSION_STATUS work? If yes, I'd prefer that.
It *is* a session level status variable, so I would not use GLOBAL_STATUS, unless necessary.

Reply:
Got it, updated it to `information_schema.SESSION_STATUS`. Thanks